mne/source_space.py

```diff
@@ -1426,12 +1432,17 @@ def setup_volume_source_space(subject, fname=None, pos=5.0, mri=None,
     """
     subjects_dir = get_subjects_dir(subjects_dir)
 
+    if fname is not None or overwrite is not None:
+        raise ValueError("Parameters 'fname' and 'overwrite' are deprecated. "
+                         "Use mne.write_source_spaces instead.")
```

Comment:
Another option would be to just remove them without deprecation (as a bug fix), since previously these params were not used.